pypsa/linopf.py

```diff
@@ -483,9 +553,23 @@ def define_global_constraints(n, sns):
         3. transmission_expansion_cost_limit
             Use this to set a limit for line expansion costs. Possible carriers
             are 'AC' and 'DC'
+        4. tech_capacity_expansion_limit
+            Use this to se a limit for the summed capacitiy of a carrier (e.g.
+            'onwind') for each investment period at choosen nodes. This limit
```

Comment:
```suggestion
            'onwind') for each investment period at chosen buses. This limit
```
buses would be more consistent with PyPSA nomenclature